content/articles/introduction-to-gui-using-tkinter-in-python/index.md

```diff
@@ -0,0 +1,162 @@
+
+You can download a python-oriented code editor, most preferably [PyCharm](https://www.jetbrains.com/pycharm/download/) since it is student-oriented, friendly and easy to use.
+
+### What are widgets?
+Widgets are the building blocks of graphical user interface programming, and they are used to view data or solicit input from the user. Here are a few examples of commonly used widgets.
+
+- `Frame`: The frame widget is a container that organizes and borders other widgets.
+- `Label`: The label widget is used to display text on the screen.
+- `Button`: The button widget can contain text and can perform an action when clicked.
+- `Entry`: The entry widget displays a single-line text field for accepting user input.
+- `Text`: A text entry widget allows you to enter multiline text.
+
+### How to build a password generator app using Tkinter
+A  password generator is a software that accepts input from a random or pseudo-random string or number generator and uses it to generate a password for the user.
+
+A password generator is a program that creates passwords depending on the rules the user select in order to build a strong and unpredictable password for multiple accounts. The password generator program generates a unique and random password for users, assisting them in creating a strong password with increased security.
+
+Online hacking is growing more common these days, resulting to passwords and login information being compromised on a regular basis. If your passwords fall into the wrong hands, it can be highly inconvenient, and can even result to the loss of money or sensitive personal information.
+
+Unfortunately, the majority of stolen passwords are simple and easy to guess. Names, birthdays, hobbies, and favorite pets don't make good passwords, and using them is a recipe for disaster. Password generators come very handy in this situation.
+
+For personal or corporate usage, the best password generators make it simple to create strong passwords that are difficult to guess or crack.
+
+Let us see how we can build a password generator.
+
+Let's start by importing Tkinter.
+
+```Python
+from tkinter import *
+from tkinter.ttk import Combobox
+import random
+```
+
+The `tkinter.ttk` module gives the user access to the Tkinter widget set and allows the user to import `Combobox`. A combobox is a graphical user interface component that combines a drop-down box, list box, or allows the user to select input.
+
+```Python
+screen = Tk()
+screen.title("Password Generator")
+screen.geometry('600x400')
+screen.configure(background ="bisque")
+``` 
+
+A root window is created using the `Tk` class. The `Tk()` function assists in the creation of this GUI window, and provides numerous options such as setting the title and the geometry of the GUI window. The `geometry()` method is one of the many methods provided by Tkinter. We'll be working on the backdrop color of the Tkinter GUI window in this line of code.
```

Comment:
When you say `We'll be working on the backdrop color of the Tkinter GUI window in this line of code`, which line are you talking of? There are lots of lines that can be interpreted as `this line`.

Try something like `We change the background color using the screen.configure() method.`